.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -118,7 +118,7 @@ on:
           Suggested variables: for E2E tests - LIT_FILTER, LIT_FILTER_OUT.
           LIT_OPTS won't work as we redefine it as part of this workflow.
 
-          For SYCL CTS - CTS_TEST_ARGS that will be passed as an argument to the
+          For SYCL CTS - CTS_TESTS_TO_BUILD that will be passed as an argument to the
           test_all executable.
```

Comment:
"...as an argument to 'ninja'" or something like that.

Reply:
I thought I've fixed this...